packages/contracts-bedrock/test/kontrol/deployment/DeploymentSummary.t.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Target contract dependencies
+import { L2OutputOracle } from "src/L1/L2OutputOracle.sol";
+import { SystemConfig } from "src/L1/SystemConfig.sol";
+import { SuperchainConfig } from "src/L1/SuperchainConfig.sol";
+import { OptimismPortal } from "src/L1/OptimismPortal.sol";
+import { DeploymentSummary } from "../proofs/utils/DeploymentSummary.sol";
+import { OptimismPortal_Test } from "test/L1/OptimismPortal.t.sol";
+
+/// @dev Contract testing the deployment summary correctness
+contract DeploymentSummary_Test is DeploymentSummary, OptimismPortal_Test {
+    /// @notice super.setUp is not called on purpose
+    function setUp() public override {
+        // Recreate Deployment Summary state changes
+        DeploymentSummary deploymentSummary = new DeploymentSummary();
+        deploymentSummary.recreateDeployment();
+
+        // Set summary addresses
+        optimismPortal = OptimismPortal(payable(OptimismPortalProxyAddress));
+        superchainConfig = SuperchainConfig(SuperchainConfigProxyAddress);
+        l2OutputOracle = L2OutputOracle(L2OutputOracleProxyAddress);
+        systemConfig = SystemConfig(SystemConfigProxyAddress);
+
+        // Set up utilized addresses
+        depositor = makeAddr("depositor");
+        alice = makeAddr("alice");
+        bob = makeAddr("bob");
+        vm.deal(alice, 10000 ether);
+        vm.deal(bob, 10000 ether);
+    }
+
+    /// @dev Skips the first line of `super.test_constructor_succeeds` because
+    ///      we're not exercising the `Deploy` logic in these tests. However,
+    ///      the remaining assertions of the test are important to check
+    function test_constructor_succeeds() external override {
+        // address guardian = deploy.cfg().superchainConfigGuardian();
+        address guardian = superchainConfig.guardian();
+        assertEq(address(optimismPortal.L2_ORACLE()), address(l2OutputOracle));
+        assertEq(address(optimismPortal.l2Oracle()), address(l2OutputOracle));
+        assertEq(optimismPortal.GUARDIAN(), guardian);
+        assertEq(optimismPortal.guardian(), guardian);
+        assertEq(optimismPortal.l2Sender(), 0x000000000000000000000000000000000000dEaD);
+        assertEq(optimismPortal.paused(), false);
+    }
+
+    /// @notice This test is skipped because `KontrolDeployment` doesn't initialize
+    ///         the L2OutputOracle
+    function test_simple_isOutputFinalized_succeeds() external override {
+        vm.skip(true);
+    }
+
+    /// @notice This test is skipped because `KontrolDeployment` doesn't initialize
+    ///         the L2OutputOracle
+    function test_isOutputFinalized_succeeds() external override {
+        vm.skip(true);
+    }
```

Comment:
@tynes @maurelian Do you have any preference to skipping these tests vs. having an empty implementation so they are marked as passing? These are the first skipped tests in the codebase, and they'll be permanent skips, so I lean towards an empty implementation (but still with a comment) so it doesn't seem like we have a tech debt item to unskip the tests

Reply:
Why are these necessary if they are always going to be skipped?

Reply:
Yes, indeed. `l2OutputOracle` is not initialized because it is not needed to run the proof. If it needed to be initialized in order to then initialize any proof-related contracts, it would have been initialized too

Reply:
I don't have particularly strong opinions here. I would be for having an empty implementation + comment

Reply:
The thing with the empty implementation is that it will always have these tests succeed, but no logic is being exercised. I thought marking them as explicitly skipped would be more informational than having them vacuously successful. But this is not a strong opinion either, just the reason why `vm.skip` is there in the first place. It is true that having them as skipped might falsely signal that there's pending work

Reply:
Let’s go with empty implementation + comment since mark and I both have a weak preference towards that